### PR TITLE
 copterhack2022 - Drone observe autonomous car - Dragon&Tanker

### PR DESCRIPTION
**Pull request by uml4**

### CODE REVIEW

File: docs/en/drone_observe_autonomous_car.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Dragon&Tanker
+
+[CopterHack-2022](copterhack2022.md), team **Dragon&Tanker**.
+
+## Team information
+
+The list of team members:
+
+(Describe the team: full name, contacts (e-mail/Telegram username), role in the team).
```
Comment (okalachev) [inline docs/en/drone_observe_autonomous_car.md:9]:
```suggestion
```

File: docs/en/drone_observe_autonomous_car.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Vo Duc Hieu, voduchieu42@gmail.com, programmer.
+* Bui Son, bhhson@gmail.com, programmer.
+
+## Project description
+
+### Project idea
+
+We will build a product which has a drone and an autonomous car for spying or delivering things 
+
+
```
Comment (okalachev) [inline docs/en/drone_observe_autonomous_car.md:22]:
```suggestion
```

File: docs/en/drone_observe_autonomous_car.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+### The potential outcomes
+
+We have already finished the robot car and be able to build the map for autonomous moving with ROS Navigation, SLAM 
+
+We will try to build some features that make a communication between the robot car and Clover Drone. The Clover will observe robot car path, if there is any obstacle on the way of robot, Clover will inform to robot car to re-create a new path.
+
+<img src="../assets/Dragon&Tanker.png" >
```
Comment (okalachev) [inline docs/en/drone_observe_autonomous_car.md:29]: Please, don't use capital letters and special characters in file names.

**Review by okalachev — CHANGES_REQUESTED** [review]
Hi! Please, check my suggestions and markdownlint's output: https://github.com/CopterExpress/clover/pull/388/checks?check_run_id=3609738409#step:5:4.

Shouldn't the article file be named drone_observe**s**_autonomous_car.md? Or you can name your article your team's name, for simplicity (but without capital letters and special characters).

**Comment by uml4** [comment]
Thank you for your review. I have update the file  name and the markdown

**Comment by okalachev** [comment]
Please no capital letters and special characters in file names, and please check this: https://github.com/CopterExpress/clover/pull/388/checks?check_run_id=3618929536#step:5:4.

**Comment by okalachev** [comment]
I fixed some issues to comply with application requirements.

**Comment by SeliverstovaE** [comment]
Dear friend,

Thank you for your interest in the CopterHack 2022 contest. Your application has successfully passed the experts’ review and was accepted for the second project stage of the competition, which will be held until February 28, 2022. This stage will be followed by monthly updates, lectures, and guidance from industry experts.

What's next?

1. Join the Telegram group with participants of the second project stage “[CopterHack 2022, participants chat](https://t.me/joinchat/1KIDu8p5FGhlYTBi )”.  
2. Every month, you're expected to prepare free-form reports on the progress of your project (update) in this draft pull request. In your report please indicate your progress, what tasks have been solved. You are encouraged to attach photos and videos that demonstrate the results of the work done within the month.
3. Send the link to the update to the participants chat in the Telegram group.

In response to the message, you will receive feedback from the competition jury.

There is no specific date for sending the report, so you can send the report between the beginning and the end of the month.

Monthly updates become mandatory from November 2021. But as soon as you are in the telegram group, you can already send updates and receive feedback from the jury.

We are interested in helping you to complete your project as much as possible by the final stage. We wish you success!

Always in touch! Your CopterHack team.

**Comment by SeliverstovaE** [comment]
Dear Dragon&Tanker team,
We are closing pull request for your project as you didn’t get to the final round of the competition. 

You are welcome to submit your article to the clover documentation as an clover platform-based project. In order to do this please create a new pull request or make a comment to the existing one.

We wish you success in implementing your project and invite you to the next events.

**Pull request closed without merging**